BitwardenShared/UI/Auth/Extensions/AuthRouter+Redirects.swift

```diff
@@ -363,14 +361,6 @@ extension AuthRouter {
                     return .landingSoftLoggedOut(email: activeAccount.profile.email)
                 }
 
-                let hasMasterPassword = activeAccount.profile.userDecryptionOptions?.hasMasterPassword ?? true
-
-                if !hasMasterPassword {
-                    let biometricUnlockStatus = try await services.biometricsRepository.getBiometricUnlockStatus()
-                    if case .available(_, true, false) = biometricUnlockStatus {
-                        return .enterpriseSingleSignOn(email: activeAccount.profile.email)
```

Comment:
🤔 We're good with not navigating to the enterprise SSO screen here because if the biometrics fails (because the user changed their settings), it will drop to the login screen, which will give them an option to re-login with SSO?

Reply:
Correct. That's a good point though, in the case of a biometrics failure (and if you don't have a MP or PIN), I think you would still remain on the vault unlock screen. We don't have logic that would log out you in that case. Should I add that?

Reply:
Can't we just show the unlock with biometrics button in the screen when the user doesn't have MP and has biometrics configured? So they can try again and if they fail 5 times then it'd go through the logic of automatic logout because of reiterative failed attempts.

Reply:
> Can't we just show the unlock with biometrics button in the screen when the user doesn't have MP and has biometrics configured? So they can try again and if they fail 5 times then it'd go through the logic of automatic logout because of reiterative failed attempts.

Once biometrics fails because the key doesn't exist, we disable biometrics and remove the biometrics button. iOS also doesn't prompt you for Face ID in this case since the key has been removed. So even if we keep the biometrics button visible, you would tap it and it would look like nothing happened. Until you eventually get logged out, so we might as well just logout from the start.

Reply:
I added a change to log the user out if biometrics fails because the key doesn't exist, and the user doesn't have a MP or PIN. 4698b86592adddeca142e9e9b2b0a97a0794ab89